### DISCUSSION TITLE
week start on monday

### DISCUSSION THREAD

**Discussion by Olifair** [freeplane/freeplane#1687] [Help] [answered]
In my version (French computer), when I open the calendar to select a reminder date, the week starts with Sunday. 
I have had a look in the preferences section "behaviour" / "Data formatting'" : the locale for formats is "France", which would imply a week start on monday. 
I have not found a preference where I can change the week-start directly (did I miss it ? I have had a look in all preferences sections)
But should the week-start not be taken from the locale for formats ? 

Oliver

**Comment by EdoFro** [comment, marked as answer]
hi @Olifair, 
I created a [Pull Request](https://github.com/freeplane/freeplane/pull/1691) that should help.

BR

edo

**Reply by Olifair** [reply]
Great, thanks. So this means that this change will be in the next FreePlane version ?

**Reply by EdoFro** [reply]
I do not know. The change has yet to be reviewed and must meet the necessary quality standard, but we have already taken a first step.

I have been testing it in my computer these days and have not had any problems yet.

**Reply by dpolivaev** [reply]
I added a new option "Use Operating System Regional Settings" and even set it to true by default.
Although I have not accepted the Pull Request by @EdoFro he played an important role because the request convinced me to improve Freeplane in this regard.

To come in the next preview.

**Reply by Olifair** [reply]
Great, thanks
